video_store.py

```diff
@@ -0,0 +1,159 @@
+import requests
+
+class VideoStore:
+    def __init__(self, url="http://localhost:5000"):
+        self.url = url
+
+
+    def check_out(self, customer_id, video_id):
+        request_params = {
+            "customer_id": customer_id,
+            "video_id": video_id,
+        }
+        response = requests.post(self.url+"/rentals/check-out", json=request_params)
+        
+        if not response:
+            return f"=== Error: Invalid ID ==="
```

Comment:
You should check for more specific errors, since things like being out of inventory can also cause this problem.